packages/geofire/package.json

```diff
@@ -42,15 +42,15 @@
     "geofire-common": "5.2.0"
   },
   "peerDependencies": {
-    "firebase": "^2.4.0 || 3.x.x || 4.x.x || 5.x.x || 6.x.x || 7.x.x || 8.x.x"
+    "firebase": "9.x.x"
```

Comment:
I'm still a bit sad about losing so much compatibility, but I guess having a `/compat` path might be a bridge too far here? Folks on older/compat versions of the JS SDK should stick to the older geofire-js release, since that hasn't changed significantly in years anyway. Should we mention that in the README?

Reply:
Added it to the migration guide here: https://github.com/thatfiredev/geofire-js/commit/654fb00b2ce520c0786c9a532f26c225802aa455